etc/di.xml

```diff
@@ -1075,6 +1075,8 @@
                 <item name="payment/adyen_apple_pay/merchant_identifier_test" xsi:type="string">1</item>
                 <item name="payment/adyen_apple_pay/merchant_identifier_live" xsi:type="string">1</item>
                 <item name="payment/adyen_google_pay/merchant_identifier" xsi:type="string">1</item>
+                <item name="payment/adyen_abstract/notification_hmac_key_test" xsi:type="string">1</item>
```

Comment:
default should be turned off

Reply:
this value should just say that the hmac key is a "sensitive" type, right?

Reply:
yes, this is just to separate these config fields as sensitive ones

Reply:
@AlexandrosMor this one needs to be 1 because it is sensitive config